### DISCUSSION TITLE
REST OData support for  $filter contains, startswith & endswith

### DISCUSSION THREAD

**Discussion by pwjohnty** [Azure/data-api-builder#2306] [Q&A]
First time posting and new to Azure DAB. 

TL;DR: Successfully set-up DAB in a container app with connection to Azure SQL Database. Successfully able to call REST API endpoint for entities within .NET MAUI application. Unable to find support/documentation for utilising OData parameters beyond the built in basic filter operations. 

Use case: Looking to use the additional $filter query parameters such as "contains", "startswith" and "endswith", for example:

`https://endpoint.com/api/table?$filter=contains(CompanyName,'freds')`

Query: is this currently supported in the REST API model within DAB? Or is there an acknowledged/best practice workaround? Currently using v1.1.7.

Appericate all help/direction!

**Comment by seantleonard** [comment]
Thank you for posting, @pwjohnty. We are tracking this ask via #936
